build-infrastructure/codebuild-devbuild-stack.yml

```diff
@@ -216,25 +216,235 @@ Resources:
         # so they have to be listed separately
         FilterGroups:
```

Comment:
This might be a good opportunity to remove these hardcoded FilterGroup entries completely, from both the existing jobs and your new ones. My understanding is that these filter groups are actually managed by an internal script, and it probably isn't good to have these github account IDs hardcoded in here (especially since 2 of these 3 people are no longer on the agent team).

Reply:
Great point. I wondered if that was outdated. The hardcoded account ID's have been removed.